LOSTANDFOUND.md

```diff
@@ -173,3 +173,14 @@ Reason for removal:
 - its only user within the RIOT code base `gnrc_ipv6_ext` was reworked in
   f671a87fe2c539c3aecd595ae03fa4f6f209d042 so it is not needed anymore.
 - the function was deprecated in f2760c033c5f332be076b25aa212aca4007c3d65
+
+### boards/wsn430-v1_3b, boards/wsn430-v1_4 [4511eebbf897e1d3ea04235c8385276c75f0c3b9]
```

Comment:
That commit hash does not seem to point anywhere. You also used multiple commits, so multiple commits should be here.

Reply:
In fact, I would prefer to provide this in a follow-up and have only commit that points to the merge commits of this PR.

Reply:
:+1: then lets do it like that, but please provide the follow-up as a depending PR before merging.

Reply:
Done: #14495